okd/src/microshift-okd-multi-build.Containerfile

```diff
@@ -0,0 +1,56 @@
+FROM quay.io/centos-bootc/centos-bootc:stream9 as builder
+
+	
+ARG OKD_REPO=quay.io/okd/scos-release
+ARG OKD_VERSION_TAG=4.17.0-0.okd-scos-2024-08-21-100712
+ARG REPO_DIR=/src/_output/rpmbuild/RPMS/
+ENV USER=microshift
+ENV HOME=/microshift
+ENV GOPATH=/microshift
+ENV GOMODCACHE=/microshift/.cache
+
+# Adding non-root user for building microshift
+run useradd -m -s /bin/bash -p "$(openssl passwd -1 redhat)" microshift
```

Comment:
This might generate a prodsec warning after we merge this code. We're not allowed to have hardcoded passwords in the repo.